samples/cpp/benchmark_app/infer_request_wrap.hpp

```diff
@@ -150,11 +190,15 @@ class InferRequestsQueue final {
     void put_idle_request(size_t id,
                           size_t lat_group_id,
                           const double latency,
+                          const InferenceResult& result,
                           const std::exception_ptr& ptr = nullptr) {
         std::unique_lock<std::mutex> lock(_mutex);
         if (ptr) {
             inferenceException = ptr;
         } else {
+            if (_result_dump) {
+                _result_dump->compare_and_save_result(result.input_images, result.output_tensors);
```

Comment:
Dumping results in parallel with inference on some lightweight topologies can be a bottleneck. Not sure that you will get what you want in this case. Requests will stuck in queue due to saving them.

Reply:
Dumping results and performance test should not be done at the same time. I will test how much the result dump affects the performance.